build.gradle

```diff
@@ -3,10 +3,11 @@
 
 buildscript {
     ext {
-        support_version = '28.0.0'
+        androidx_version = '1.0.0'
+        material_version = '1.0.0'
         picasso_version = '2.71828'
         glide_version = '4.9.0'
-        constraint_version = '1.1.3'
+        constraint_version = '1.1.2'
```

Comment:
Why use version 1.1.2?
When I use 1.1.3 version, it works well.
Is there any special reason?

Reply:
nope, changed back to 1.1.3 👍